### ISSUE TITLE
brew install soapyuhd not working

### ISSUE THREAD

**Issue by wonderingStars** [pothosware/homebrew-pothos#55]
Installing soapyuhd from pothosware/pothos
==> cmake .. -DUHD_ROOT='.'
==> make install
Last 15 lines from /Users/newuser/Library/Logs/Homebrew/soapyuhd/02.make:
    res_source_info src_info{res_source_info::USER, instance};
                    ^
/opt/homebrew/include/uhd/rfnoc/res_source_info.hpp:29:5: note: 'res_source_info' has been explicitly marked deleted here
    res_source_info() = delete;
    ^
fatal error: too many errors emitted, stopping now [-ferror-limit=]
20 errors generated.
make[2]: *** [CMakeFiles/uhdSupport.dir/SoapyUHDDevice.cpp.o] Error 1
make[1]: *** [CMakeFiles/uhdSupport.dir/all] Error 2
make[1]: *** Waiting for unfinished jobs....
[ 80%] Linking CXX shared module libsoapySupport.so
/opt/homebrew/Cellar/cmake/3.22.2/bin/cmake -E cmake_link_script CMakeFiles/soapySupport.dir/link.txt --verbose=1
/opt/homebrew/Library/Homebrew/shims/mac/super/clang++ -O3 -DNDEBUG -arch arm64 -isysroot /Library/Developer/CommandLineTools/SDKs/MacOSX12.sdk -bundle -Wl,-headerpad_max_install_names -o libsoapySupport.so CMakeFiles/soapySupport.dir/UHDSoapyDevice.cpp.o  /opt/homebrew/lib/libuhd.dylib /opt/homebrew/lib/libSoapySDR.0.8.1.dylib /opt/homebrew/lib/libboost_thread-mt.dylib /opt/homebrew/lib/libboost_system-mt.dylib -pthread -flat_namespace 
[ 80%] Built target soapySupport
make: *** [all] Error 2

If reporting this issue please do so at (not Homebrew/brew or Homebrew/core):
  https://github.com/pothosware/homebrew-pothos/issues

lastest macOS  : Montaray MacBook 13 M1

**Comment by ncorgan** [comment]
@guruofquality We should make a new SoapyUHD tag. This looks like the C++14 flag issue we fixed, which hasn’t made it into a release.

**Comment by wonderingStars** [comment]
did I install the wrong one

**Comment by ncorgan** [comment]
No, the fix for that problem hasn’t made it into a release yet.

**Comment by wonderingStars** [comment]
ok do you know any other software I could use for Mac that supports ettus b200

**Comment by guruofquality** [comment]
just FYI passing --head to brew should make it build in the meantime

**Comment by wonderingStars** [comment]
I try to build brew
same

**Comment by wonderingStars** [comment]
> just FYI passing --head to brew should make it build in the meantime

hi I have managed to install it but can't get it to recognise the b200 it all works on windows but getting work on mac os is the worst thing in the world

**Comment by metasim** [comment]
> just FYI passing --head to brew should make it build in the meantime

You're a lifesaver!! 🙇
